client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -96,15 +71,8 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 	// Ensure that the selected site is fetched, if available. This is used for event tracking purposes.
 	// See https://github.com/Automattic/wp-calypso/pull/82981.
 	const selectedSite = useSelector( ( state ) => site && getSite( state, siteSlugOrId ) );
-	const isRequestingSelectedSite = useSelector(
-		( state ) => site && isRequestingSite( state, siteSlugOrId )
-	);
-
-	// Short-circuit this if the site slug or ID is not available.
-	const hasRequestedSelectedSite = siteSlugOrId
-		? !! selectedSite && ! isRequestingSelectedSite
-		: true;
 
+	// TODO: Check if it we still need to check the selected site to preload the steps.
```

Comment:
Yes, we do. This is just a [measure](https://github.com/Automattic/wp-calypso/pull/91008) to defer preloading steps after the more important loading of the site. We never really needed it.

Reply:
Thanks for the context! No changes applied to this logic!